cmd/soroban-rpc/internal/ledgerbucketwindow/ledgerbucketwindow.go

```diff
@@ -21,14 +20,18 @@ type LedgerBucket[T any] struct {
 	BucketContent        T
 }
 
+// DefaultEventLedgerRetentionWindow represents the max number of ledgers we would like to keep
+// an incoming event in memory
+const DefaultEventLedgerRetentionWindow = 17280
```

Comment:
I know you added this to the help text, but it'd be nice to note in the comment here that this is 24hr worth of ledgers (assuming 5s/ledger close times)

Reply:
done.